src/libs/portfolio/getOnchainBalances.ts

```diff
@@ -194,11 +195,15 @@ export async function getNFTs(
     simulationToken?.collectibles?.forEach((newCollectible: bigint) => {
       if (!token.collectibles.includes(newCollectible)) receiving.push(newCollectible)
     })
+    const tokenError =
```

Comment:
there should be an explanation of what's going on there

Reply:
reverted this addition altogether as the try catch in the nft getter was the right approach